### PR TITLE
BMI calculator example

### PR DESCRIPTION
**Pull request by lutaii**
**What does this implement/fix? Explain your changes**
BMI Calculator example

**Does this close any currently open issues?**
Issue #103 

+ **Where has this been tested?**
  - iPhone 14 Pro
    
  
  - Pixel 3A
    

+ **Target Platform:**
iOS/Android

+ **Walkthrough:**

### CODE REVIEW

**Comment by lutaii** [comment]
Hey @nisrulz. 
Please review my pull request.

File: bmi_calculator/lib/calculator_page.dart [hunk 1]
```diff
@@ -0,0 +1,400 @@
+import 'package:bmi_calculator/result_page.dart';
```
Comment (nisrulz) [inline bmi_calculator/lib/calculator_page.dart:1]: This file is unreadable as it is a giant nested Page. Break the nested widgets into their own properties or functions, so your final Calculator page looks more like

```
Calculator page
   ScaffoldWidget
     BodyWidget
       Content
         Gender Selector Widget
         Counter Widget
         Weight Widget
         Height Widget
```
where each widget is declared separately i.e Gender Selector Widget is encapsulated in a function or rather written in a new file as a Stateful Widget.

File: bmi_calculator/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    sdk: flutter
+  flutter_lints: ^2.0.0
+
+flutter:
+  uses-material-design: true
+
+  module:
+    androidX: true
+    androidPackage: com.example.bmi_calculator
+    iosBundleIdentifier: com.example.bmiCalculator
```
Comment (nisrulz) [inline bmi_calculator/pubspec.yaml:26]: Refactor package name to follow standard as set in this repo. [Mentioned in readme](https://github.com/nisrulz/flutter-examples#pull-requests)

**Review by nisrulz — COMMENTED** [review]
Some comments to address. This PR needs some work.

Additionally, also note 
- You need to add a link back in readme index
- Update the package name

**Comment by lutaii** [comment]
Hey @nisrulz. Can you review my changes once again?

**Comment by lutaii** [comment]
Hey @nisrulz 
Can I merge this pr?

**Comment by nisrulz** [comment]
Hi @lutaii I still need to take a look at this PR. I'll try to merge open PRs in the next 2-3 days.

**Pull request merged**